### ISSUE TITLE
Missing Navigation Submenu Chevron Button Missing `aria-expanded` Attribute on load

### ISSUE THREAD

**Issue by jeryj** [WordPress/gutenberg#50752]
### Description
The Interactivity API for the Navigation block doesn't include the `aria-expanded` attribute for the submenu chevron button so the state of the submenu being expanded/collapsed is not announced to screen readers. 

### Step-by-step reproduction instructions
1. Turn on your screen reader of choice
2. Have a navigation with a submenu
3. Go to the frontend of the site
4. Use the keyboard to place focus on the submenu chevron icon
5. The state of collapsed or expanded is not announced

On the non-interactivity API version of the navigation, the aria-expanded state is set on load so the state of "collapsed" is announced.

**Comment by SantosGuillamot** [comment]
Thanks a lot for reporting this 🙂  I created [this pull request](https://github.com/WordPress/gutenberg/pull/50786) to handle that among other accessibility issues.

**Comment by luisherranz** [comment]
Fixed by https://github.com/WordPress/gutenberg/pull/50786.
